### DISCUSSION TITLE
v4.3.4.1084 introduced breaking change on MqttClientOptionsBuilder not mentioned in release notes

### DISCUSSION THREAD

**Discussion by ramonsmits** [dotnet/MQTTnet#2003] [General]
The `Server` and `Port` properties were removed and it seems that happened in the following PR:

- https://github.com/dotnet/MQTTnet/pull/1919

No `[Obsolete]` on properties to guide developers on how to migrate....

For other developers you need to migrated to use the `RemoteEndpoint` property:

I have a need to force `AddressFamily.InterNetwork` so I could not use the `string,string` overload of `WithTcpServer`:
```c#
x.AddressFamily = System.Net.Sockets.AddressFamily.InterNetwork;
x.RemoteEndpoint = new DnsEndPoint(Server.Host, Server.Port);
```

**Comment by chkr1011** [comment]
That change mas made by accident so I restored these properties.

Please try the following version from myget: https://www.myget.org/feed/mqttnet/package/nuget/MQTTnet/4.3.5.1137

It restores the properties.
